### PR TITLE
[fix][doc] Correcting spelling mistakes

### PR DESCRIPTION
**Pull request by daziz**
### Modifications

Correcting spelling mistakes

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/daziz/pulsar

### CODE REVIEW

**Review by tisonkun — COMMENTED** [review]
It seems you submit similar patches to correct typos. Could you go through them module by module so that we can batch patches to reduce CI burden?

**Comment by tisonkun** [comment]
@daziz Good to know. I'm open to these contributions. Thank you!

The reason for batching similar minor changes is to reduce CI burden and best engineering practice. There is no reason to separate typo fixes found on the same day into multiple PRs - Please help save reviewers' time so we can handle more contributions.

File: pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaUtils.java [hunk 1]
```diff
@@ -210,8 +210,7 @@ public static String jsonifySchemaInfo(SchemaInfo schemaInfo) {
     }
 
     /**
-     * Jsonify the schema info with verison.
-     *
+     * Jsonify the schema info with version
```
[the remark this answers is not in the document]
Comment (daziz) [reply pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaUtils.java:213]: ver**is**on -> version

[the remark this answers is not in the document]
Comment (tisonkun) [reply pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaUtils.java:213]: Oops, you are right. Then let's revert other unrelated changes.

```suggestion
     * Jsonify the schema info with version.

```

File: pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaUtils.java [hunk 2]
```diff
@@ -210,8 +210,8 @@ public static String jsonifySchemaInfo(SchemaInfo schemaInfo) {
     }
 
     /**
-     * Jsonify the schema info with verison.
-     *
+     * Jsonify the schema info with version.
+
```
Comment (tisonkun) [inline pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaUtils.java:214]:
```suggestion
     *
```

**Review by tisonkun — APPROVED** [review]

**Review by RobertIndie — APPROVED** [review]

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**